pkgs/applications/version-management/git-crecord/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchFromGitHub, pythonPackages }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "git-crecord-${version}";
+  version = "20161216.0";
+
+  src = fetchFromGitHub {
+    owner = "andrewshadura";
+    repo = "git-crecord";
+    rev = version;
+    sha256 = "0v3y90zi43myyi4k7q3892dcrbyi9dn2q6xgk12nw9db9zil269i";
+  };
+
+  buildInput = with pythonPackages; [ docutils ];
```

Comment:
I tried that as well, but it won't work. I guess `buildInputs` would suffice here, as `docutils` is only a built-time dependency, as far as I can see...